package.json

```diff
@@ -7,6 +7,6 @@
   "author": "Adrian Gierakowski <adrian.gierakowski@deversifi.com>",
   "license": "MIT",
   "dependencies": {
-    "debug": "^4.1.1"
+    "safe-json-stringify": "^1.2.0"
```

Comment:
I removed that `.enabled` at all, as we dont seem to use it, and I created simple pattern to account for `DEBUG` env var to allow logging all levels with `DEBUG=*` and specific levels in isolation e.g. `DEBUG=WARN`

by default we would log following levels (basically all without DEBUG level)
`[ LOG, WARN, ERROR, EMERGENCY ]`

Reply:
I'm sorry, I'm not going to approve this PR if we don't get feature parity with existing logger. It's very simple the achieve this: bring back the debug module and use it's `.enabled` prop to check if you should log or not (even though you don's use it for actual logging).

Reply:
btw. if we use `debug` module (and/or `DEBUG` env var) we'll need the `dvf:` prefix as there are many other modules in the JS ecosystem using `debug` and if we don't prefix we might accidentally enable logging in one of the libraries we used. Keeping it backward compatible will also save you time since you will not need to update env vars in existing services which use `dvf-logger`